### ISSUE TITLE
Progress bar won't disappear if page loading is stopped

### ISSUE THREAD

**Issue by Regnander** [luben/load-progress-bar#5]
Press the Stop button (or Esc) while a page is loading and the progress bar will remain on-screen.

![screenshot-2018-1-4 luben load-progress-bar 2](https://user-images.githubusercontent.com/17113053/34578311-bd4a0610-f184-11e7-8902-6c32e8ecc14d.png)
.

**Comment by luben** [comment]
Yes, when you hit the stop on a page it disables JS execution and all other activities. The add-on is itself a small JS injected in the page that draws the progress bar so it will stop updating in this case.

I will research if there is a way to detect and handle this case.

**Comment by luben** [comment]
There is the "onabort" and "oncancel" events, i.e.:

https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onabort
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/GlobalEventHandlers.oncancel

but looks they are never fired - at least when I tried to handle them there was no effect, the handler was never called.
